Apps/GatewayApi/src/Services/UserProfileService.cs

```diff
@@ -145,7 +145,6 @@ public RequestResult<UserProfileModel> GetUserProfile(string hdid, DateTime jwtA
             }
 
             userProfile.HasTermsOfServiceUpdated = retVal.Payload?.TermsOfServiceId != termsOfServiceResult.ResourcePayload?.Id;
-            userProfile.AcceptedTermsOfService = retVal.Payload?.TermsOfServiceId != Guid.Empty;
```

Comment:
This goes along with the code above but this is a special scenario where if the Guid is empty it has to be not accepted and otherwise the code flow would indicate a true value.

Reply:
See my reply to line 126 change.